lightblue/src/test/java/org/esbtools/eventhandler/lightblue/BulkLightblueRequesterTest.java

```diff
@@ -433,7 +439,43 @@ public void shouldCacheBothGuaranteedRequestsAndTriedRequestsTogetherAndSendToLi
             throw e;
         }
     }
-
+    
+    @Test
+    public void shouldBeOrderedIfRequested() throws Exception {
+
+        LightblueClient mockClient = Mockito.mock(LightblueClient.class);
+        LightblueBulkDataResponse lightblueDataResponse = Mockito.mock(LightblueBulkDataResponse.class);
+        when(mockClient.bulkData(any(DataBulkRequest.class))).thenReturn(lightblueDataResponse);
+
+        BulkLightblueRequester orderedRequester = new BulkLightblueRequester(mockClient, true);
+        orderedRequester.request(new DataFindRequest("foo"), new DataFindRequest("bar")).get();
+        Mockito.verify(mockClient).bulkData(Mockito.argThat(new ArgumentMatcher<DataBulkRequest>() {
+            @Override
+            public boolean matches(Object argument) {
+                assertTrue(((DataBulkRequest) argument).isOrdered());
+                return true;
```

Comment:
.verify will handle failing the test for us or not, so I think you can just return true or false here without an assert inside the matcher. In other words, we want to "verify that the bulk request is ordered" so we want the matcher to only return true if the request is ordered (that's what we're verifying), without side effects.